tests/ballerina-tools-integration-test/src/test/java/org/ballerinalang/test/run/BalRunFunctionNegativeTestCase.java

```diff
@@ -37,15 +37,24 @@
  */
 public class BalRunFunctionNegativeTestCase extends BaseTest {
 
-    private String fileName = "test_entry_function.bal";
-    private String filePath = (new File("src/test/resources/run/file/" + fileName)).getAbsolutePath();
+    private String sourceRootPath = "src/test/resources/run/file/";
 
     @Test
-    public void testEmptyEntryFunctionName() throws BallerinaTestException {
-        String sourceArg = filePath + ":";
-        LogLeecher errLogLeecher = new LogLeecher("ballerina: expected function name after final ':'",
+    public void testInvalidSourceArg() throws BallerinaTestException {
+        String sourceArg = (new File(sourceRootPath + "test_entry_function.bal")).getAbsolutePath() + ":";
+        LogLeecher errLogLeecher = new LogLeecher("error: no ballerina source files found in module " + sourceArg,
                 LeecherType.ERROR);
         balClient.runMain(sourceArg, new LogLeecher[]{errLogLeecher});
         errLogLeecher.waitForText(2000);
     }
+
+    @Test
```

Comment:
Shall we add test descriptions to these test cases so that it'll be clear to understand what's being actually tested?